docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+name: ultrasinger
+services:
+    ultrasinger:
+        deploy:
+            resources:
+                reservations:
+                    devices:
+                        - driver: nvidia
```

Comment:
Does this work on machinges without nvidia gpus?

Reply:
I tried it on my gpu less NAS 😄 

> UltraSinger Starting Error response from daemon: could not select device driver "nvidia" with capabilities: [[gpu]]

Reply:
i have made a second one for non gpu users

Reply:
its been tested with and without gpu support. it now works with both.
do note there are new two docker-compses
one for gpu and one for cpu